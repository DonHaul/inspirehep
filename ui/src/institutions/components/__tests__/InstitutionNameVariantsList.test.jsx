import React from 'react';
import { render } from '@testing-library/react';
import { fromJS } from 'immutable';
import InstitutionsNameVariantsList from '../InstitutionNameVariantsList';

describe('InstitutionNameVariantsList', () => {
  it('renders', () => {
    const nameVariants = fromJS([
      {
        value: 'Name1',
      },
      {
        value: 'Name2',
      },
    ]);
    const { asFragment } = render(
      <InstitutionsNameVariantsList nameVariants={nameVariants} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
