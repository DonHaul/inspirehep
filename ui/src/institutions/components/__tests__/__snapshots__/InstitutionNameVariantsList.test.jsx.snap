// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstitutionNameVariantsList renders 1`] = `
<DocumentFragment>
  <div
    class="__InlineList__"
    data-testid="inline-data-list"
  >
    <span
      class=""
    >
      Name Variants: 
    </span>
    <ul>
      <li>
        <span>
          Name1
        </span>
        , 
      </li>
      <li>
        <span>
          Name2
        </span>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
